tests/rbe_repo/generate_confs.yaml

```diff
@@ -0,0 +1,101 @@
+# Steps in this file use toolchain configs produced by rbe_autoconf**
```

Comment:
Add some individual comments for each RBE build about what it's testing?

Reply:
done